stor/cli.py

```diff
@@ -72,10 +72,14 @@
 Direct file transfer between OBS services or within one OBS service (server-side copy)
 is not yet supported.
 """
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
```

Comment:
Note that most of the calls to `str` in this module are pointless (since they're `'%s' % str(something)`).